### PR TITLE
refactor(ai): fix lint warnings

### PR DESCRIPTION
**Pull request by rickstaa**
**What does this pull request do? Explain your changes. (required)**

This commit fixes some linting errors to clean up the code a bit.

**Specific updates (required)**

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Checklist:**

- [ ] Read the [contribution guide](./CONTRIBUTING.md)
- [ ] `make` runs successfully
- [ ] All tests in `./test.sh` pass
- [ ] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: server/ai_worker_test.go [hunk 1]
```diff
@@ -29,11 +28,6 @@ import (
 
 func TestRemoteAIWorker_Error(t *testing.T) {
 	httpc := &http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}
-	//test request
```
Comment (rickstaa) [inline server/ai_worker_test.go:32]: @ad-astra-video I'm probablly overlooking something but this appears to be unused.

File: net/lp_rpc.pb.go [hunk 2]
```diff
@@ -1,24 +1,24 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// versions:
-// 	protoc-gen-go v1.33.0
-// 	protoc        v3.12.4
 // source: net/lp_rpc.proto
 
 package net
 
 import (
-	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
```
Comment (rickstaa) [inline net/lp_rpc.pb.go]: @leszko not sure why so much changes in this file between protoc versions 🤔.

File: server/ai_worker_test.go (hunk 1 above)
Comment (ad-astra-video) [reply server/ai_worker_test.go:32]: Believe that is correct.  Moved creating jobs to `createAIJob` and forgot to delete this out.

Comment (rickstaa) [reply server/ai_worker_test.go:32]: No problem 👍🏻.

File: net/lp_rpc.pb.go (hunk 2 above)
Comment (rickstaa) [reply net/lp_rpc.pb.go]: This happend due to a local setting on my system reverted this 👍🏻.

**Review by leszko — APPROVED** [review]

**Pull request merged**